website/content/en/preview/tasks/configuration.md

```diff
@@ -17,7 +17,7 @@ There are two main configuration mechanisms that can be used to configure Karpen
 | AWS_DEFAULT_INSTANCE_PROFILE | \-\-aws-default-instance-profile | The default instance profile to use when provisioning nodes in AWS|
 | AWS_ENABLE_POD_ENI | \-\-aws-enable-pod-eni | If true then instances that support pod ENI will report a vpc.amazonaws.com/pod-eni resource (default = false)|
 | AWS_ENI_LIMITED_POD_DENSITY | \-\-aws-eni-limited-pod-density | Indicates whether new nodes should use ENI-based pod density. DEPRECATED: Use `.spec.kubeletConfiguration.maxPods` to set pod density on a per-provisioner basis (default = true)|
-| AWS_ISOLATED_VPC | \-\-aws-isolated-vpc | If true then assume we can't reach AWS services which don't have a VPC endpoint (default = false)|
+| AWS_ISOLATED_VPC | \-\-aws-isolated-vpc | If true then assume we can't reach AWS services which don't have a VPC endpoint (default = false). This also has the effect of disabling look-ups to the AWS pricing endpoint.|
```

Comment:
This will get replaced when we do a new `docs-gen` right? This either needs to be added to the source that does the generation or the flag help directly

Reply:
Yep, so we would want to make the change [here](https://github.com/aws/karpenter/blob/main/pkg/utils/options/options.go#L81) and then run `make doc-gen`, @chrisnegus .

Reply:
Okay. I'll take care of that.

Reply:
Thanks for pointing this out. Done. Please check to make sure I got it right.